config/coverband.rb

```diff
@@ -44,39 +44,19 @@
 
 # 3.x configuration
 Coverband.configure do |config|
-  config.root = Dir.pwd
-  config.store = if ENV['REDIS_URL']
-                   Coverband::Adapters::RedisStore.new(Redis.new(url: ENV['REDIS_URL']))
-                 else
-                   Coverband::Adapters::RedisStore.new(Redis.new)
-                 end
-
-  config.ignore = %w[vendor .erb$ .slim$]
-  # add paths that deploy to that might be different than local dev root path
-  config.root_paths = []
-
-  # reporting frequency
-  # if you are debugging changes to coverband I recommend setting to 100.0
-  # otherwise it is find to report slowly over time with less performance impact
-  # with Coverage collector coverage is ALWAYS captured this is how frequently
-  # it is reported to your back end store.
-  # NOTE: the demo site sends data for EVERY request so one can see the effects
-  # do not send 100.0 in production!
-  config.reporting_frequency = Rails.env.production? ? 10.0 : 100.0
-  config.logger              = Rails.logger
-
   # configure S3 integration
   config.s3_bucket = 'coverband-demo'
```

Comment:
We should also have solid defaults for all of these s3 params. For rails, we could use the application name as part of the bucket even:

```ruby
Rails.application.class.to_s.split("::").first.downcase
```

Reply:
I think given folks have to setup the bucket and bucket permissions outside of Coverband, this one is worth leaving more explicit... a default might make folks assume the gem can handle creating and configuring a bucket. So I think it is best to skip this.